it/AssetsControllerISpec.scala

```diff
@@ -61,24 +62,65 @@ class AssetsControllerISpec extends IntegrationSpecBase {
     )
   )
 
+  val wsdlOperationsForFileNamesV2 = Map(
+    "BusinessActivityService/ICS/ReferralManagementBAS/V2/CCN2.Service.Customs.EU.ICS.ReferralManagementBAS_2.0.0_CCN2_2.0.0.wsdl" -> List(
+      "IE4Q04requestAdditionalInformation",
+      "IE4R02provideAdditionalInformation",
+      "IE4Q05requestHRCM",
+      "IE4R03provideHRCMResult"
+    ),
+    "BusinessActivityService/ICS/RiskAnalysisOrchestrationBAS/V2/CCN2.Service.Customs.EU.ICS.RiskAnalysisOrchestrationBAS_2.0.0_CCN2_2.0.0.wsdl" -> List(
+      "IE4N03notifyERiskAnalysisHit",
+      "IE4S01submitEScreeningAssessment",
+      "IE4S02submitRiskAnalysisResult",
+      "IE4S02updateERiskAnalysisResult",
+      "IE4S01updateEScreeningResult"
+    ),
+    "BusinessActivityService/ICS/ReferralManagementBAS/V2/CCN2.Service.Customs.EU.ICS.ReferralManagementBAS_2.0.0_CCN2_2.0.0.wsdl" -> List(
+      "IE4Q04requestAdditionalInformation",
+      "IE4R02provideAdditionalInformation",
+      "IE4Q05requestHRCM",
+      "IE4R03provideHRCMResult"
+    ),
```

Comment:
This should be checking the CRErrorNotificationBas or removed, as we've already got test cases above for the ReferralManagementBAS